sdk/face/azure-ai-vision-face/src/main/java/module-info.java

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) TypeSpec Code Generator.
+
+module com.azure.ai.vision.face {
+    requires transitive com.azure.core;
+    exports com.azure.ai.vision.face;
+    exports com.azure.ai.vision.face.models;
+    opens com.azure.ai.vision.face.implementation.models to com.azure.core, com.fasterxml.jackson.databind;
+    opens com.azure.ai.vision.face.models to com.azure.core, com.fasterxml.jackson.databind;
+    opens com.azure.ai.vision.face.implementation.models to com.azure.core;
+    opens com.azure.ai.vision.face.models to com.azure.core;
```

Comment:
remove this 2 lines (it should be covered by the 2 line above)

Reply:
Fixed.